src/blocks/mrc_call_python_function.ts

```diff
@@ -127,6 +128,11 @@ const CALL_PYTHON_FUNCTION = {
           tooltip = 'Calls the function ' + className + '.' + functionName + '.';
           break;
         }
+        case FUNCTION_KIND_INSTANCE_WITHIN: {
+          const functionName = this.getFieldValue(pythonUtils.FIELD_FUNCTION_NAME);
+          tooltip = 'Calls the method ' + functionName + '.';
+          break;
+        }
         default:
           throw new Error('mrcVarKind must be "module", "static", "constructor", or "instance".')
```

Comment:
Does this need to say or instance_within?

Reply:
Good catch.